unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/BodyShapeController.cs

```diff
@@ -140,16 +147,34 @@ protected override void PrepareWearable(GameObject assetContainer)
                 lowerBodyRenderer = r;
             else if (parentName.Contains("feet"))
                 feetRenderer = r;
+            else if (parentName.Contains("head"))
+                headRenderer = r;
+            else if (parentName.Contains("eyebrows"))
+                eyebrowsRenderer = r;
+            else if (parentName.Contains("eyes"))
+                eyesRenderer = r;
+            else if (parentName.Contains("mouth"))
+                mouthRenderer = r;
```

Comment:
We shouldn't be using `WearableLiterals` here?

Reply:
On a second thought, maybe not because we are handling renderers here, not the wearable model. And the `ubody` `lbody` don't match the literals. Maybe we can use some constants.

Reply:
We could and it would work, but I dont know... 

In `WearableLiterals` we have well defined literals like Categories, BodyShapeIds, Rarities and such. In this case we are just (in a really ugly way xD) checking the renderer's parent name to now what part of the bodyshape it belongs to. It's just a sort of convention we arranged with the artists that is not actually defined anywhere else like the rest of literals.

We could benefit for having consts but maybe not in `WearableLiterals` to avoid confusion? I dont have strong feelings against the proposed solution though

Reply:
>We could benefit for having consts but maybe not in WearableLiterals to avoid confusion? I dont have strong feelings against the proposed solution though

Maybe this could work, but it'd only help if we need to make this ugly check somewhere else.